com.system76.Popsicle.yaml

```diff
@@ -0,0 +1,35 @@
+app-id: com.system76.Popsicle
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+  env:
+    CARGO_HOME: /run/build/popsicle/cargo
+finish-args:
+- --share=ipc
+- --socket=fallback-x11
+- --socket=wayland
+- --device=dri
+- --device=all
+- --filesystem=home
```

Comment:
I wonder why this permission is necessary, GtkNativeFileChooser can't be used here?

Reply:
Right. I don't think migrating Popsicle to using `GtkNativeFileChooser` would pose any difficulties...

Until very recently recently Popsicle couldn't run in a sandbox, and I don't believe it's been tested on Windows or Mac. So there wasn't any difference, but it looks like switching to `GtkNativeFileChooser` would be a good idea now.

Reply:
Not a blocker, but it would be nice to land that in the future. Merged

Reply:
Yep. That, and potentially using UDisks2 to find block devices so `--device=all` isn't needed.

Reply:
Ah didn't catch this one